src/pytorch_lightning/strategies/launchers/multiprocessing.py

```diff
@@ -164,6 +165,16 @@ def _collect_rank_zero_results(self, trainer: "pl.Trainer", results: Any) -> Opt
 
         return _WorkerOutput(best_model_path, weights_path, trainer.state, results, extra)
 
+    def _check_torchdistx_support(self) -> None:
+        if self._start_method == "spawn":
+            from pytorch_lightning.utilities.meta import _is_deferred
+
+            if _is_deferred(self._strategy.lightning_module):
```

Comment:
I think you need to handle `is_deferred(None)`? For Lite the `lightning_module` reference would be None